src/ibek/ioc.py

```diff
@@ -50,6 +53,13 @@ def __post_init__(self):
             assert inst_id not in id_to_entity, f"Already got an instance {inst_id}"
             id_to_entity[inst_id] = self
 
+        # create a context dictionary for use in Jinja expansion of this Entity
+        context = asdict(self)
+        context["__utils__"] = self.__utils__
+        # todo jinja expand all values with context
+        self.__context__ = context
+        # then pass __context__ to jinja template in render.py
+
```

Comment:
I understand this bit

Reply:
So I changed this bit to pull all of the attributes using get_type_hints. This then includes any class variables added to the dataclass.